README.md

```diff
@@ -103,6 +103,8 @@ The projects that still use ruby 2.7 should eventually be converted to Ruby 3.0
 
 Note: this includes dor-services-app and sdr-api in addition to cocina level2 updates.
 
+**[Turn off Google Books](https://sul-gbooks-prod.stanford.edu/features) when deploying to production.**
```

Comment:
Can this say "if a cocina-model diference is in the change set" or "so that we don't see problems when sdr-api rejects a mis-matched cocina version"?